rice-namenode/source/main.cc

```diff
@@ -37,16 +37,17 @@ static inline int parse_cmdline_options(
     int argc,
     char *argv[],
     int *port,
-    int *verbosity
+    int *verbosity,
+    char *node_policy
```

Comment:
Please update the documentation with this new param `node_policy`

Reply:
Added